packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -553,7 +553,7 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
         case Axis.vertical:
           _gestureRecognizers = <Type, GestureRecognizerFactory>{
             VerticalDragGestureRecognizer: GestureRecognizerFactoryWithHandlers<VerticalDragGestureRecognizer>(
-              () => VerticalDragGestureRecognizer(),
+              () => VerticalDragGestureRecognizer(rejectMousePointers: true),
```

Comment:
Does the developer have a way to override this behavior? I wonder whether this should be a theme-thing. cc @HansMuller

Reply:
ScrollConfiguration seems like a reasonable choice - assuming it is only created for lists and things? Like draggable sheets need to still be draggable, but that doesn't use a scrollable correct?